canlink/mcap.go

```diff
@@ -0,0 +1,151 @@
+package canlink
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/foxglove/mcap/go/mcap"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"os"
+	"strconv"
+)
+
+type Mcap struct {
```

Comment:
Add comments above all exports. Comments should be full sentences and end with a period.